### DISCUSSION TITLE
RSSOwlnix 2.8.0-beta's startup slows down gradually

### DISCUSSION THREAD

**Discussion by sam0thius** [Xyrio/RSSOwlnix#115] [Q&A] [answered]
Hi there, 

Have noticed that with RSSOwlnix 2.8.0-beta the program's startup time gradually slows down as items in the database keep mounting up. The startup time goes from what uses to be few seconds to around 20 seconds. Usually it takes couple months for the startup to slow down, and running the program's internal cleanup does not resolve this. By my findings the only way to restore the startup speed of the program back to original is to delete the entire C:\Users\profile\ .rssowlnix-beta folder. Initially tested just deleting the database rssowl.db file, but that alone does not affect the slow startup time.  

Any idea what might be causing this slowdown? Thx!

**Comment by Xyrio** [comment]
difficult to know.  
- how many tabs are open before you close?
- is this log growing rapidly, does it add many errors on each startup? `%userprofile%\.rssowlnix-beta\.metadata\.log`
- there is a bug that makes the followin file grow, try deleting it: `%userprofile%\.rssowlnix-beta\.metadata\.plugins\org.eclipse.e4.workbench\workbench.xmi`
- you could check if any other file grows constantly just from restarting rssowlnix over and over. either copy the profile folder and compare with a program like kdiff3 or just make a list of files and sizes like: `dir /S /ON > list.txt` and compare those.

hmm seems like the workbench.xmi is the cause. #55

**Reply by sam0thius** [reply]
Unfortunately I went and Emptied the Windows Recycle Bin yesterday after deleting the entire .rssowlnix-beta folder, but I'll keep monitoring the file sizes for a while and report back once the slowdown starts again.

**Comment by sam0thius** [comment, marked as answer]
Managed to extract rssowlnix-beta folder from a month or so old system backup image and it seems to confirm the workbench.xmi as the culprit (see below file size comparisons.) Once the new Rssowlnix instance starts to slow down, i'll try first to delete only workbench.xmi file to resolve the issue. 

366 KB workbench.xmi // old file
  71 KB workbench.xmi // new file
